src/graphnet/data/dataconverter.py

```diff
@@ -566,3 +570,27 @@ def _skip_frame(self, frame: "icetray.I3Frame") -> bool:
         if frame["I3EventHeader"].sub_event_stream == "NullSplit":
             return True
         return False
+
+    def _filter_mask(
+        self, frame: "icetray.I3Frame", I3filters: List[str]
```

Comment:
You can get rid of the codeclimate error if you let _filter_mask access the i3filters inside the function, instead of passing them as argument.